### ISSUE TITLE
Is NASA using Remix 🛰️💿?

### ISSUE THREAD

**Issue by kentcdodds** [tachgsfc/gcn.nasa.gov#48]
Just wondering.

Also, we're going to put together a users page for remix and I'd love to include NASA on that page if possible.

**Comment by lpsinger** [comment]
Yes we are! But this web page isn't quite ready for prime time yet. I have to ask about the NASA logo on the Remix site.

**Comment by lpsinger** [comment]
Is the users page live now? Can I see it for some context?

**Comment by kentcdodds** [comment]
I'll ping you when we've got one. Could be a couple months out

**Comment by kentcdodds** [comment]
Just the logo most likely. We may choose to feature specific company projects that allow us to
